staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -158,18 +156,19 @@ func TestUniformFlows(t *testing.T) {
 
 	clk, counter := clock.NewFakeEventClock(now, 0, nil)
 	qsf := NewQueueSetFactory(clk, counter)
+	dealer, err := shufflesharding.NewDealer(8, 3)
```

Comment:
I'll shut up about the Dealer now, but moving its construction out here doesn't seem like an improvement?

Reply:
The real motivation for the move is in the config controller.  I am willing to tolerate this test code.